src/main/java/core/basesyntax/ShopService.java

```diff
@@ -0,0 +1,5 @@
+import java.util.List;
+
+public interface ShopService {
```

Comment:
The interface name `ShopService` could be more specific to its purpose. For instance, if it's meant to handle operations related to fruit transactions, a name like `FruitTransactionService` might be more descriptive.